src/test/java/core/basesyntax/service/impl/FileWriterServiceImplTest.java

```diff
@@ -0,0 +1,73 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.service.FileWriterService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FileWriterServiceImplTest {
+    private static final String OUTPUT_TEST_PATH = "src/test/resources/outputTest.csv";
+    private static final String INPUT_TEST_PATH = "src/test/resources/inputTest.csv";
+    private static final StringBuilder builder = new StringBuilder();
+    private static FileWriterService fileWriterService;
+
+    @BeforeClass
+    public static void beforeClass() {
+        fileWriterService = new FileWriterServiceImpl();
+        builder.append("type,fruit,amount")
+                .append(System.lineSeparator()).append("b,apple,90")
+                .append(System.lineSeparator()).append("b,banana,30")
+                .append(System.lineSeparator()).append("s,banana,60")
+                .append(System.lineSeparator()).append("s,banana,40")
+                .append(System.lineSeparator()).append("p,banana,25")
+                .append(System.lineSeparator()).append("r,apple,10")
+                .append(System.lineSeparator()).append("p,apple,20")
+                .append(System.lineSeparator()).append("p,banana,15");
+    }

```

Comment:
```suggestion
    private static String data;
    private static FileWriterService fileWriterService;

    @BeforeClass
    public static void beforeClass() {
        fileWriterService = new FileWriterServiceImpl();
        data = new StringBuilder().append("type,fruit,amount")
                .append(System.lineSeparator()).append("b,apple,90")
                .append(System.lineSeparator()).append("b,banana,30")
                .append(System.lineSeparator()).append("s,banana,60")
                .append(System.lineSeparator()).append("s,banana,40")
                .append(System.lineSeparator()).append("p,banana,25")
                .append(System.lineSeparator()).append("r,apple,10")
                .append(System.lineSeparator()).append("p,apple,20")
                .append(System.lineSeparator()).append("p,banana,15").toString();
    }
```
better create a string